docs/README.md

```diff
@@ -25,7 +25,7 @@ class Status extends \Konekt\Enum\Enum {
 
```

Comment:
We will also need to maintain two parallel versions of the [Documentation](https://artkonekt.github.io/enum/#/), something like the [PHPUnit Documentation](https://phpunit.readthedocs.io/en/8.0/). (screenshot below)

Enum's doc is built with [docsify](https://docsify.js.org/) + github pages. Are you aware of any versioning solution for docsify?

![image](https://user-images.githubusercontent.com/1162360/54497273-f02bc380-48f8-11e9-9c25-66839fbd6f9f.png)

Reply:
Docsify supports [Multiple covers](https://docsify.js.org/#/cover?id=multiple-covers). It is useful for languages or versions.

Reply:
Cool. Problem 1/2 solved. Now just have to figure out how to integrate that with github-pages.